packages/cozy-client/src/CozyClient.js

```diff
@@ -94,13 +94,15 @@ class CozyClient {
     this.handleTokenRefresh = this.handleTokenRefresh.bind(this)
 
     this.createClient()
+    const stackClient = this.getStackClient()
+    stackClient.on('error', (...args) => this.emit('error', ...args))
```

Comment:
I see no `off()` for this, is it expected ?

Reply:
We have no public method to replace or remove the stack client. It should be live as long as the main client is live.